jaas/src/main/java/com/griddynamics/jagger/jaas/service/impl/TestEnvironmentServiceImpl.java

```diff
@@ -100,7 +94,7 @@ else if (!isEqualCollection(testEnvToUpdate.getLoadScenarios(), newTestEnv.getLo
         }
 
         if (testEnvToUpdate.getRunningLoadScenario() != newTestEnv.getRunningLoadScenario()) {
-            testEnvToUpdate.setRunningLoadScenario(getNewRunningLoadScenario(newTestEnv, testEnvToUpdate));
+            testEnvToUpdate.setRunningLoadScenario(newTestEnv.getRunningLoadScenario());
```

Comment:
This code may produce persistence exceptions if newTestEnv.getRunningLoadScenario() is not persisted yet.
To omit such situations I'd propose you to check if env has this load scenario and if no - add it.